frontend/src/lang/modules/en.ts

```diff
@@ -195,6 +195,8 @@ const message = {
             dbName: `This field mustn't start with underscore character and must consist of English, numbers, and "_" characters with a length of 1-64.`,
             imageName:
                 'This field must consist of English, numbers, ":", "@", "/", ".", "-", and "_" characters with a length of 1-256.',
+            composeName:
+                'Supports non-special characters at the beginning, lowercase letters, numbers, - and _, length 1-256',
             volumeName:
                 'This field must consist of English, numbers, ".", "-", and "_" characters with a length of 2-30.',
             supervisorName:
```

Comment:
The changes appear to correct some validation conditions for specific fields in an object literal:

1. **`dbName`**: The message now mentions that underscores cannot be used in the string, which was previously incorrect.
2. **`composeName`**: This field is new and appears to allow for more flexible names with support for non-special characters at the start.

While these corrections enhance clarity regarding allowed character sets and constraints, they might not have been necessary if the previous version already covered those aspects adequately. It would also help to ensure consistency across similar fields by applying a standard pattern or using a regex for validation.